src/EditorFeatures/CSharpTest/AddParameter/AddParameterTests.cs

```diff
@@ -2587,5 +2589,57 @@ void TestFunc()
     void MyFunc(BaseClass param1, int newparam) { }
 }");
         }
+
+        [WorkItem(44271, "https://github.com/dotnet/roslyn/issues/44271")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsAddParameter)]
+        public async Task TopLevelStatement()
+        {
+            await TestMissingAsync(@"
+<Workspace>
+    <Project Language=""C#"" LanguageVersion=""Preview"" AssemblyName=""Assembly1"" CommonReferences=""true"">
+        <Document>
+[|local|](1, 2, 3);
+
+void local(int x, int y)
+{
+}
+        </Document>
+    </Project>
+</Workspace>",
+            new TestParameters(parseOptions: TestOptions.Regular.WithLanguageVersion(LanguageVersion.CSharp8)));
```

Comment:
i'm confused by the Project xml specifying the lang version and the test specifying something different.

Reply:
Yes, sorry about that. 
I can remove the `.WithLanguageVersion` (tests would still work). 
But since it's the main change I'd make to address feedback and the PR has already gone through CI, I'm tempted to leave as-is until the scenarios get properly fixed. Let me know if that's okay.